src/components/Form/FormProvider.tsx

```diff
@@ -251,8 +251,16 @@ function FormProvider(
         },
         [inputValues],
     );
+
+    const resetError = useCallback(() => {
```

Comment:
```suggestion
    const resetErrors = useCallback(() => {
```
I think `resetErrors` will be a good fit. Also, merge main.